openapi-cli/src/test/java/io/ballerina/openapi/generators/openapi/ServiceDeclarationNodesTests.java

```diff
@@ -49,23 +49,23 @@ public void setup() throws IOException {
     public void multipleServiceWithSameAbsolute() throws IOException {
         Path ballerinaFilePath = RES_DIR.resolve("multiple_services.bal");
         executedMethod(ballerinaFilePath, "multiple_service_01.yaml", "hello_openapi.yaml",
-                "hello-");
+                "hello_");
     }
 
     @Test(description = "Multiple services with absolute path as '/'. ")
     public void multipleServiceWithOutAbsolute() throws IOException {
         Path ballerinaFilePath = RES_DIR.resolve("multiple_services_without_base_path.bal");
         executedMethod(ballerinaFilePath, "multiple_service_02.yaml",
```

Comment:
Should this be `executedMethod` or `executeMethod`? I suppose the latter one sounds correct.

Reply:
Yeah I will change with `executeMethod` since the function's purpose is to execute the method.